File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Mar.java
package org.firstinspires.ftc.teamcode.util;

public class Mar {
    //@TODO lol fix this shit
    private double time;
    private double start;

    public Mar() {
        time = 0;
        start = 0;
    }
    public void start() {
        start = System.currentTimeMillis();
    }

    public void stop() {
        time = System.currentTimeMillis() - start;
    }

    public double getTime() {
        return time;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/Mar.kt
package org.firstinspires.ftc.teamcode.util

class Mar {
    var start = 0.0
    var time = System.currentTimeMillis() - start
        private set

    fun start() { start = System.currentTimeMillis().toDouble() }
    fun stop() { time = System.currentTimeMillis() - start }
}